backend/organization/views/project_views.py

```diff
@@ -23,14 +37,102 @@ def get_serializer_class(self):
         return ProjectMinimalSerializer
 
 
-class ProjectAPIView(ListAPIView):
-    lookup_field = 'pk'
-    serializer_class = ProjectSerializer
-    pagination_class = None
+"""
+Note: I am splitting organization project creation and personal project creation into
+two separate APIs. It makes more sense do two APIs so that we can manage user accessibility
+by project or organization member roles.
+"""
+
+
+class OrganizationProjectsView(APIView):
```

Comment:
Shouldn't this be called something like `OrganizationsCreateProjectView`? I think the name doesn't really make it clear that this view is to create a project.

Reply:
Good call. Yeah I will change the name